sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/WriteFiles.scala

```diff
@@ -52,6 +52,7 @@ case class WriteFiles(
     options: Map[String, String],
     staticPartitions: TablePartitionSpec) extends UnaryNode {
   override def output: Seq[Attribute] = child.output
+  override protected def stringArgs: Iterator[Any] = Iterator(child)
```

Comment:
why do we exclude all other information?

Reply:
but eventually we will remove these information from v1 write command, after we always do planned write.